docs/Behaviour - Client Side.md

```diff
@@ -1,15 +1,19 @@
-# PUTting Media Profiles
+# Behaviour: Client Side
 
-Before making an IS-05 connection, NMOS Controller chooses Receivers intended to be a part of this connection. If they are IS-11 compatible Receivers, NMOS Controller MUST gather their Receiver Capabilities and make such intersection of them that they can be utilized for building Media Profiles satisfying all the Receivers. After that NMOS Controller MUST PUT /media-profiles and make the IS-05 connection if succeed. After breaking this connection via IS-05, NMOS Controller is RECOMMENDED to DELETE /media-profiles.
+## Constraints of Sender
 
-If multiple Receivers have an equal Constraint Set, NMOS Controller MUST take only one of them. If they differ only in `urn:x-nmos:cap:meta:preference`, then one with the highest priority MUST be taken.
+Before making an [IS-05][IS-05] connection, NMOS Controller chooses Receivers intended to be a part of this connection. If these Receivers are managed with IS-11, NMOS Controller MUST gather their Receiver Capabilities and make such intersection of them that they can be utilized for building Constraints satisfying all the Receivers. After that NMOS Controller MUST `PUT /constraints` and make the IS-05 connection if succeed. After breaking this connection via IS-05, NMOS Controller is RECOMMENDED to `DELETE /constraints`.
 
-# Dynamic format changes on Sender
+If multiple Receivers have equal Constraint Sets, NMOS Controller MUST take only one of them. If they differ only in `urn:x-nmos:cap:meta:preference`, then one with the highest priority MUST be taken.
 
-## Changes which break Media Profiles
+## Dynamic format changes on Sender
 
-NMOS Controller MUST track Sender's `subscription`'s `active` property during the IS-05 connection. If it became `false` unexpectedly, NMOS Controller SHOULD GET `/media-profiles` to figure out the reason of breaking the connection. If the request returns `205 Reset Content`, NMOS Controller SHOULD either disconnect all the Receivers or PUT `/media-profiles` once again and re-enable Sender via IS-05.
+### Changes which break Constraints
 
-## Changes which do not break Media Profiles
+NMOS Controller MUST track Sender's `subscription`'s `active` property during the IS-05 connection. If it became `false` unexpectedly, NMOS Controller SHOULD `GET /constraints` to figure out the reason of breaking the connection. If the request returns `205 Reset Content`, NMOS Controller SHOULD either disconnect all the Receivers or `PUT /constraints` once again and re-enable Sender via IS-05.
```

Comment:
I believe the sender subscription is meant to be used in unicast mode only as currently defined in IS-04:
https://specs.amwa.tv/is-04/releases/v1.3.1/APIs/schemas/with-refs/sender.html

Reply:
I raised this topic in https://github.com/AMWA-TV/nmos-discovery-registration/issues/170 because https://specs.amwa.tv/is-04/releases/v1.3.1/docs/4.3._Behaviour_-_Nodes.html#senders explicitly dictates different. The schema is clarified in https://github.com/AMWA-TV/nmos-discovery-registration/pull/171 though it's not in IS-04 1.3.1.

Also, we refer to this property in "Behaviour: Server Side" describing the same behaviour from the Node's point of view.